eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -132,7 +132,7 @@ jobs:
       - template: ../variables/globals.yml
 
     pool:
-      vmImage: 'windows-2019'
```

Comment:
I like this workaround better than the other PR which changes the JDK version from 11 to 8.